snippets/SMTP/smtp_example.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Net;
+using System.Net.Mail;
+ 
+public class EmailTest
+{
+   public static void Main()
+   {
+        MailAddress to = new MailAddress("developers@sparkpost.com");
```

Comment:
REPLACE_WITH_YOUR_EMAIL_ADDRESS is a bit verbose in my opinion.  I'm thinking to switch this to simply 'recipient@example.com', which seems common in other email examples.  I'm also going to add comments about sparkpostbox.com